src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/UnapprovedLicenceController.kt

```diff
@@ -51,4 +54,35 @@ class UnapprovedLicenceController(private val unapprovedLicenceService: Unapprov
   fun getEditedLicencesUnapprovedByCrd(): List<UnapprovedLicence> {
     return unapprovedLicenceService.getEditedLicencesNotReApprovedByCrd()
   }
+
+  @PostMapping(value = ["/notify-probation-of-unapproved-licences"])
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
+  @ResponseBody
+  @Operation(
+    summary = "Send an email to probation practitioner of any previously approved licences that have been edited but not re-approved by prisoners release date",
+    description = "Send email to probation practioner. Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
+    security = [SecurityRequirement(name = "ROLE_SYSTEM_USER"), SecurityRequirement(name = "ROLE_CVL_ADMIN")],
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Emails sent",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = UnapprovedLicence::class))],
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorised, requires a valid Oauth2 token",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Forbidden, requires an appropriate role",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      )
+    ]
+  )
+  fun notifyProbationOfUnapprovedLicences(@Valid @RequestBody unapprovedLicenceEmailData: List<UnapprovedLicence>) {
```

Comment:
had a question around whether you need this list here? was there a reason you didnt just retrieve the licences in `unapprovedLicenceService.sendEmailsToProbationPractitioner` rather than have them do the round trip- retrieving from the endpoint and the posting them back to another endpoint?